pkg/internal/common/helper.go

```diff
@@ -44,6 +52,7 @@ var ChainMetadataMap = map[int64]types.ChainMetadata{
 		ELDelegationManagerAddress:  "0x39053D51B77DC0d36036Fc1fCc8Cb819df8Ef37A",
 		ELAVSDirectoryAddress:       "0x135dda560e946695d6f155dacafc6f1f25c1f5af",
 		ELRewardsCoordinatorAddress: "0x7750d328b314EfFa365A0402CcfD489B80B0adda",
+		ELPermissionManagerAddress:  "",
```

Comment:
Note: unclear on the process to populate this.

Reply:
we will hardcode it when the deployment happens and we can also add that as a flag so that someone/us in other envs can override. In general I want to refactor the network and env to make it better. but that can wait till slashing PR is merged to dev. that will be a all command refactor